src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -90911,84 +90878,92 @@ INSERT INTO tab_event_conditions VALUES(5938,'EVENT','ParticleBinDissolvedEventU
+    container_name text,
+    transport_type text,
+    membrane       text,
+    constraint tab_known_transporters_containers_pk
+        primary key (gene, species, container_type, container_name),
+    constraint tab_known_transporters_containers_tab_known_transporters_gene_species_fk
+        foreign key (gene, species) references tab_known_transporters (gene, species)
+);
+INSERT INTO tab_known_transporters_containers VALUES('ABCC3','Human','COMPARTMENT','Pericentral','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC3','Human','COMPARTMENT','Periportal','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC4','Human','COMPARTMENT','Pericentral','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC4','Human','COMPARTMENT','Periportal','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC6','Human','COMPARTMENT','Pericentral','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC6','Human','COMPARTMENT','Periportal','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('SLC22A9','Human','COMPARTMENT','Pericentral','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('SLC22A9','Human','COMPARTMENT','Periportal','Efflux','Basolateral');
+CREATE TABLE tab_transport_directions (transport_direction text CONSTRAINT tab_transport_directions_pk PRIMARY KEY, display_name text NOT NULL, description text NOT NULL, icon text DEFAULT Influx);
+INSERT INTO tab_transport_directions VALUES('InfluxPlasmaToInterstitial','Influx Plasma to Interstitial','Influx transport from blood plasma to interstitial space across endothelial barrier','Influx');
+INSERT INTO tab_transport_directions VALUES('EffluxInterstitialToPlasma','Efflux Interstitial to Plasma','Efflux transport from interstitial space to blood plasma across endothelial barrier','Efflux');
+INSERT INTO tab_transport_directions VALUES('InfluxPlasmaToBloodCells','Influx Plasma to Blood Cells','Influx transport from blood plasma to blood cells across blood cell membrane','Influx');
+INSERT INTO tab_transport_directions VALUES('EffluxBloodCellsToPlasma','Efflux Blood Cells to Plasma','Efflux transport from blood cells to blood plasma across blood cell membrane','Efflux');
+INSERT INTO tab_transport_directions VALUES('InfluxInterstitialToIntracellular','Influx Interstitial to Intracellular','Influx transport from interstitial into intracellular space across cell membrane','Influx');
+INSERT INTO tab_transport_directions VALUES('EffluxIntracellularToInterstitial','Efflux Intracellular to Interstitial','Efflux transport from intracellular into interstitial space across cell membrane','Efflux');
+INSERT INTO tab_transport_directions VALUES('InfluxLumenToMucosaIntracellular','Influx Lumen To Mucosa/Intracellular','Active uptake from gastrointestinal lumen into mucosal cells','Influx');

```

Comment:
`'Influx Lumen To Mucosa/Intracellular'`
Replace "To" with "to"

Reply:
also stuff from before. If you look at the dump, this is a dump o the db

Reply:
Same. Old error, but I didn't noticed it before